docs/notary.md

```diff
@@ -45,9 +45,19 @@ Chain dump without notary service does not have predefined network map.
 Therefore, you need to wait about 5 minutes until new epoch tick with updated
 network map.
 
-All helper commands such as `make tick.epoch` or `make update.epoch_duration`
-are designed as multisignature transactions, so you can't use them without
-notary.
+
+4. Enable helper commands
+
+To enable helper commands such as `make tick.epoch` or `make update.epoch_duration`
+make sure to export non-empty `NEOFS_NOTARY_DISABLED` environment variable. 
+```
+$ export NEOFS_NOTARY_DISABLED=1
+```
+
+Use `unset` command to return it back.
```

Comment:
wouldn't it be more clear(explicit) to check the value of the flag and not the existence?

Reply:
I thought about it but didn't like the idea to specify exact value in the *flag* for bash script. It can be `=1, =true, =TRUE, =True, =on, =On`.

Reply:
just wanted to say that `NEOFS_NOTARY_DISABLED=false`/`NEOFS_NOTARY_DISABLED=0` works not as somebody may expect

Reply:
This is valid concert. Any more thoughts?
/cc @aprasolova @anatoly-bogatyrev

Reply:
so my opinion is "specific action to turn the flag on" > "specific action to turn the flag off"

Reply:
> specific action to turn the flag on

It must be on by default, therefore optional flag can only disable it.